packages/datagateway-search/cypress/integration/search/datafileSearch.spec.ts

```diff
@@ -89,10 +92,10 @@ describe('Datafile search tab', () => {
       .should('not.exist');
   });
 
-  it('should link to a parent dataset', () => {
+  it.only('should link to a parent dataset', () => {
```

Comment:
This seems to have sneaked in somehow. This is my fault, I was working on this a short while ago.